tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -206,3 +206,20 @@ Given('the app {string} has been disabled', function (app) {
       'app:disable', app
     ])
 })
+
+Given('default expiration date for users is set to {int} day/days', function (days) {
+  // FIXME: make array in runOcc functon work
+  occHelper.runOcc(
```

Comment:
you can also use the step `Given the setting "shareapi_default_expire_date_user_share" of app "core" has been set to "yes"`

Reply:
Yeah, wanted to keep the steps in scenario minimal. But will use that 👍